src/main/java/de/unistuttgart/regexgamebackend/service/ConfigurationService.java

```diff
@@ -0,0 +1,85 @@
+package de.unistuttgart.regexgamebackend.service;
+
+import de.unistuttgart.regexgamebackend.data.Configuration;
+import de.unistuttgart.regexgamebackend.repository.ConfigurationRepository;
+import java.util.List;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+@Transactional
```

Comment:
Erm… Typically, only `Repository` (interaces|methods) should be annotated with `Transactional`.
In addition, I don't think it's needed here as the default methods generated by `JpaRepository` default should already be `@Transactional`?
(but please test it first)

Reply:
This has been copied from [finitequiz-backend](https://github.com/Gamify-IT/finitequiz-backend/blob/5c7457500c12cf24e90dd435649779f758d20485/src/main/java/de/unistuttgart/finitequizbackend/service/ConfigService.java#L26) so that would maybe be something we should consider across several backends globally.